packages/acs-ui-javascript-loaders/src/callWithChatCompositeLoader.ts

```diff
@@ -54,6 +54,6 @@ export const loadCallWithChatComposite = async function (
     throw new Error('Failed to find the root element');
   }
 
-  createRoot(htmlElement).render(React.createElement(CallWithChatComposite, { ...props, adapter }, null));
+  createRoot(htmlElement).render(React.createElement(CallWithChatComposite, { options: props, adapter }, null));
```

Comment:
We should change the fn signature naming.. we have args that are of type ...Props, and we have props that are of type ...Options

Reply:
We also have args, nullable element, options
What is an option vs an args?
Args are arguments which should encompass all args but for don't include two args that follow the args

Reply:
(note not related to this PR, but we should fix that naming before stable)